README.md

```diff
@@ -1,6 +1,6 @@
-# [Robot Name] - [Competition Type] Repository
+# Smorphi - [Competition Type] Repository
```

Comment:
We can replace [Competition Type] here with Learning. So it can read:

Smorphi - Learning Repository